.github/tyk_pro_values.yaml

```diff
@@ -6,44 +6,52 @@ nameOverride: ""
 fullnameOverride: ""
 
 # Used to shard the gateways. If you enable it make sure you have at least one gateway that is not sharded and also tag the APIs accordingly
-enableSharding: true
+enableSharding: false
 
 # Set to true to use Tyk as the Ingress gateway to an Istio Service Mesh
 # We apply some exceptions to the Istio IPTables so inbound calls to the
 # Gateway and Dashboard are exposed to the outside world - see the deployment templates for implementation
-enableIstioIngress: true
+enableIstioIngress: false
+
+# Master switch for enabling/disabling bootstraing job batch, role binding, role, and account service.
+bootstrap: true
 
 secrets:
-  APISecret: "CHANGEME"
+  APISecret: CHANGEME
   AdminSecret: "12345"
 
 redis:
-    shardCount: 128
-    #If you're using Bitnami Redis chart please input the correct host to your instalaltion in the field below
-    #addrs:
-    #- "tyk-redis-master.tyk.svc.cluster.local:6379"
-    # addrs:
-    #   - "redis.tyk.svc.cluster.local:6379"
-    useSSL: false
-    #If you're using Bitnami Redis chart please input your password in the field below
-    #pass: ""
+  shardCount: 128
+  # addrs:
+  #   - redis.tyk.svc.cluster.local:6379
+  useSSL: false
+#    #If you're using Bitnami Redis chart please input the correct host to your installation in the field below
+#    addrs:
+#      - tyk-redis-master.tyk.svc.cluster.local:6379
+#   #If you're using Bitnami Redis chart please input your password in the field below
+#    pass: ""
+#   #If you are using Redis cluster, enable it here.
+#    enableCluster: false
+#   #By default the database index is 0. Setting the database index is not supported with redis cluster. As such, if you have enableCluster: true, then this value should be omitted or explicitly set to 0.
+  storage:
+    database: 0
 
 mongo:
-    # mongoURL: "mongodb://mongo.tyk.svc.cluster.local:27017/tyk_analytics"
-    #If you're using Bitnami MongoDB chart please input your password below
-    #mongoURL: "mongodb://root:pass@tyk-mongo-mongodb.tyk.svc.cluster.local:27017/tyk-dashboard?authSource=admin"
-    useSSL: false
+  # mongoURL: mongodb://mongo.tyk.svc.cluster.local:27017/tyk_analytics
+  # If you're using Bitnami MongoDB chart please input your password below
+  # mongoURL: mongodb://root:pass@tyk-mongo-mongodb.tyk.svc.cluster.local:27017/tyk-dashboard?authSource=admin
+  useSSL: false
 
 mdcb:
-  enabled: true
+  enabled: false
```

Comment:
Why disable it and other components for the tests?

Reply:
I've enabled it now.